src/main/java/core/basesyntax/Shape.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public interface Shape {
+    void drawShape();
+
+    double getArea();
+
+    Figure getRandomFigure();
```

Comment:
You don't need that method in the figure, just create `getRandomFigure()` method in FugureSupplier